client/packages/common/src/intl/locales/en/common.json

```diff
@@ -155,7 +155,7 @@
   "description.fc-sell-price": "Foreign currency sell price per pack",
   "description.forecast-quantity": "Forecast Quantity to Reach Target",
   "description.initial-stock-on-hand": "Stock on hand on the first day of the program period",
-  "description.available-stock": "The customer's initial stock on hand + incoming stock + inventory adjustments - outgoing stock", 
+  "description.available-stock": "The customer's initial stock on hand + incoming stock + inventory adjustments - losses - outgoing stock", 
```

Comment:
Pendantic but inventory adjustments would be `additions - losses` already

Reply:
You are right 🤣  I love pedantic-ness